### ISSUE TITLE
`public` is used for function without access argument

### ISSUE THREAD

**Issue by mjhagen** [foundeo/toscript#9]
`<cffunction name="myFunc">` is converted to `public function myFun()` (adding `public`)

**Comment by KamasamaK** [comment]
Since `public` is the default access this is semantically consistent. I suppose the issue then might be that it's not a verbatim translation. So the issue really comes down to whether the conversion should be opinionated on best practice.

**Comment by mjhagen** [comment]
@KamasamaK, yeah I just reported it for that purpose. Couldn't think of a way this would be a problem though. Very low priority (maybe even no priority :))
